client/src/containers/Overview/index.jsx

```diff
@@ -182,57 +182,99 @@ export class Overview extends React.Component {
-                          <Accordion>
-                            <Accordion.Title
-                              active={activeIndex === `commit-${monthAndYear}`}
-                              index={`commit-${monthAndYear}`}
-                              onClick={this.handleActivityState}
-                            >
-                              <p>
-                                Created {commitCount} commit{commitCount === 1 ? '' : 's'} in {numOfRepos} repositor
-                                {numOfRepos === 1 ? 'y' : 'ies'}
-                              </p>
-                              <Octicon icon={activeIndex === `commit-${monthAndYear}` ? Fold : Unfold} />
-                            </Accordion.Title>
-                            <Accordion.Content active={activeIndex === `commit-${monthAndYear}`}>
-                              {Object.entries(monthActivityObject).map(([repoName, repoCommits]) => {
-                                const link = `${username}/${repoName}`;
-                                return (
-                                  <div key={repoName}>
-                                    <Link to={link} className={styles.activity_link}>
-                                      {username}/{repoName}
-                                    </Link>{' '}
-                                    <span>{repoCommits} commits</span>
-                                  </div>
-                                );
-                              })}
-                            </Accordion.Content>
-                          </Accordion>
-                        </div>
-                      </Container>
-                    </>
-                  );
-                })}
+              <Container className={styles.section_header}>
+                <h2>Contribution activity</h2>
+              </Container>
+              {Object.entries(userMonthActivity).length > 0 &&
+                Object.entries(userMonthActivity)
+                  .sort((prev, next) => moment(next[0]).diff(moment(prev[0])))
+                  .map(([date, monthActivityObject]) => {
+                    const monthAndYear = moment(date).format('MMMM YYYY');
+                    const commitCount =
+                      monthActivityObject.commits &&
+                      Object.keys(monthActivityObject.commits).reduce((acc, key) => {
+                        return acc + monthActivityObject.commits[key];
+                      }, 0);
+                    const numOfCreatedRepos = monthActivityObject.createdRepos.length;
+                    const numOfCommittedRepos =
+                      monthActivityObject.commits && Object.keys(monthActivityObject.commits).length;
+                    return (
+                      <Fragment key={date}>
+                        <Container key={monthAndYear} className={styles.contribution_activity}>
+                          <h3>{monthAndYear}</h3>
+                          {monthActivityObject.commits && (
```

Comment:
```suggestion
                          {Object.entries(monthActivityObject.commits).length && (
```
Пустой объект возвращает true (если monthActivityObject.commits - объект). Выше тоже нужно поправить

Reply:
monthActivityObject.commits существует, если существует хоть один коммит, т.е. этот объект существует с данными о коммитах внутри, либо его нет (если нет коммитов). Поэтому здесь проверка на length избыточна

Reply:
А, понятно, окей тогда